source/MRCommonPlugins/ViewerButtons/MRViewerSettingsPlugin.cpp

```diff
@@ -437,6 +443,55 @@ void ViewerSettingsPlugin::drawQuickAccessList_()
     }
 }
 
+void ViewerSettingsPlugin::drawSpaceMouseSettings_( float scaling )
+{
+    auto& viewerRef = Viewer::instanceRef();
+    ImVec2 windowSize( 400 * scaling, 300 * scaling );
+    ImGui::SetNextWindowPos( ImVec2( ( viewerRef.window_width - windowSize.x ) / 2.f, ( viewerRef.window_height - windowSize.y ) / 2.f ), ImGuiCond_Always );
+    ImGui::SetNextWindowSize( windowSize, ImGuiCond_Always );
+
+    if ( !ImGui::BeginModalNoAnimation( "Spacemouse settings", nullptr, ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoTitleBar ) )
+        return;
+
+    ImGui::Text( "%s", "Spacemouse settings" );
```

Comment:
mb add some tooltip that we use screen space for axes

Reply:
it seems intuitive